bidrag-regnskap/src/main/kotlin/no/nav/bidrag/regnskap/service/KonteringService.kt

```diff
@@ -28,7 +28,9 @@ class KonteringService {
         perioderForOppdragsperiode.forEachIndexed { indexPeriode, periode ->
 
             // Om det opprettes et opphør og det ikke finnes eksisterende korrigerte konteringer for perioden skal det ikke opprettes ny kontering.
-            if (oppdragsperiode.opphørendeOppdragsperiode && oppdragsperiode.beløp == BigDecimal.ZERO && !finnesKorrigerendeKonteringerForMåned(
+            if (oppdragsperiode.opphørendeOppdragsperiode &&
+                oppdragsperiode.beløp == BigDecimal.ZERO &&
```

Comment:
Ser at det her bare er en formaterings-endring, men er greit å være klar over uansett: Det å sammenlikne BigDecimal med == er ikke så trygt, i tilfelle det er to foskjellige instanser. Å sammenlikne med .equals er også skummelt fordi 0 ikke er lik 0.0, f.eks... Bør bruke .compareTo() == 0...